yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/DockerProvisioningStrategy.java

```diff
@@ -66,7 +66,8 @@ public void setEnabled(boolean enabled) {
 
                 int availableCapacity = snapshot.getAvailableExecutors() +
                         snapshot.getConnectingExecutors() +
-                        strategyState.getAdditionalPlannedCapacity();
+                        strategyState.getAdditionalPlannedCapacity() +
+                        strategyState.getPlannedCapacitySnapshot();
```

Comment:
I used this https://github.com/jenkinsci/mansion-cloud-plugin/blob/master/src/main/java/com/cloudbees/jenkins/plugins/mtslavescloud/MansionNodeProvisionerStrategy.java#L123-L124
@stephenc do you have bug in mansion?